event_routing_backends/signals.py

```diff
@@ -0,0 +1,50 @@
+import json
+import logging
+
+from django.conf import settings
+from django.dispatch import receiver
+from django_redis import get_redis_connection
+from edx_django_utils.cache.utils import get_cache_key
+from openedx_events.analytics.signals import TRACKING_EVENT_EMITTED
+from openedx_events.event_bus import get_producer
+
+from event_routing_backends.config import SEND_TRACKING_EVENT_EMITTED_SIGNAL
+
+logger = logging.getLogger(__name__)
+
+TRANSFORMED_EVENT_KEY_NAME = "transformed_events"
+
+@receiver(TRACKING_EVENT_EMITTED)
+def listen_for_tracking_event_emitted_event(sender, signal, **kwargs):
+    """
+    Publish `TRACKING_EVENT_EMITTED` events to the event bus.
+    """
+    event = kwargs['tracking_log']
+
+    ## TODO: Should we ignore events that we don't care about here or in the event routing backend config?
```

Comment:
@bmtcril Please read my comments

Reply:
It's possible to ignore events at the tracking log configuration level, which may be enough for this and is probably better for overall system performance.